CHANGELOG.next.asciidoc

```diff
@@ -20,6 +20,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Ensure dynamic template names are unique for the same field. {pull}18849[18849]
 - Remove the deprecated `xpack.monitoring.*` settings. Going forward only `monitoring.*` settings may be used. {issue}9424[9424] {pull}18608[18608]
 - Added `certificate` TLS verification mode to ignore server name mismatch. {issue}12283[12283] {pull}20293[20293]
+- Make the mage binary used by the build process in the docker container to be statically compiled. {pull}20827[20827]
```

Comment:
Maybe this belongs in the developer changelog.

Reply:
good point, changing that.